### PR TITLE
Bug fixes

### LINKED ISSUE #85: Add start page to set server url

### LINKED ISSUE #90: Adjust app title bar
The current page title bar should indicate the current navigation page

For example:  
Currently every page has `soTired` printed as title. It would be better if the titled would display `Questionnaire` when taking the questionnaire or `Self test` when testing your fatigue level.

If this is somehow confusing, please let me know and I will explain more detailed 😅

### LINKED ISSUE #94: Fix PVT test
* Disable multiple button presses during the test
* Also make sure the time is only measured on first button press
* Also add units of measurement to measured times

### LINKED ISSUE #101: Bug fixes in app user interface
- [x] change home page (scrolling)
- [x] change app behavior when app in background
- [x] change questionnaire progress apperance
- [x] change pvt workflow
- [x] change spatial span task exceptions and popups

### PR DESCRIPTION
**Pull request by carina-str**
closes #101 
closes #94 
closes #90
closes #85

### CODE REVIEW

**Review by diehmlukas — APPROVED** [review]
nice work!! :+1:  :1st_place_medal:

**Review by diehmlukas — CHANGES_REQUESTED** [review]
found bug by clicking in the textbox to set the URL.

File: app/lib/ui/modules/pvt_test/widgets/pvt_test_diff.dart [hunk 1]
```diff
@@ -9,5 +9,5 @@ class PVTTestDiff extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) =>
-      Text(diff, style: Theme.of(context).textTheme.headline2);
+      Text(diff + ' ms', style: Theme.of(context).textTheme.headline2);
```
Comment (mac641) [inline app/lib/ui/modules/pvt_test/widgets/pvt_test_diff.dart:12]:
```suggestion
      Text('$diff ms', style: Theme.of(context).textTheme.headline2);
```

**Review by diehmlukas — APPROVED** [review]
nice work again :+1:

File: app/lib/ui/modules/spatial_span_test/engine/game_state.dart [hunk 2]
```diff
@@ -42,7 +41,7 @@ class StartState extends GameState {
 
   @override
   void checkUserInteraction(int boxId) {
-    throw GameWrongStateException();
+    // throw GameWrongStateException();
```
Comment (mac641) [inline app/lib/ui/modules/spatial_span_test/engine/game_state.dart:44]: Add TODO why this is commented

File: app/lib/ui/modules/spatial_span_test/engine/game_state.dart [hunk 3]
```diff
@@ -62,7 +61,7 @@ class ShowSequenceState extends GameState {
...
```
Comment (mac641) [inline app/lib/ui/modules/spatial_span_test/engine/game_state.dart:64]: Add TODO why this is commented

File: app/lib/ui/modules/spatial_span_test/engine/game_state.dart [hunk 4]
```diff
@@ -107,7 +106,7 @@ class GameOverState extends GameState {
...
```
Comment (mac641) [inline app/lib/ui/modules/spatial_span_test/engine/game_state.dart:109]: Add TODO why this is commented

File: app/lib/ui/modules/spatial_span_test/engine/game_state.dart [hunk 5]
```diff
@@ -130,7 +129,7 @@ class GameOverState extends GameState {
 class NextLevelState extends GameState {
   @override
   void checkUserInteraction(int boxId) {
-    throw GameWrongStateException();
+    // throw GameWrongStateException();
```
Comment (mac641) [inline app/lib/ui/modules/spatial_span_test/engine/game_state.dart:132]: Add TODO why this is commented

**Review by mac641 — CHANGES_REQUESTED** [review]
Please take a look a these tiny code style improvements, I've got 👍🏿

Also, we've found a bug where there is no error thrown if the server can't be reached after entering a valid URL in the settings page - Would be great to have that fixed before merging this PR ^^

**Review by diehmlukas — APPROVED** [review]
:+1:

**Review by mac641 — APPROVED** [review]
Well done! 😊

**Pull request merged**